apps/etterlatte-behandling/src/main/kotlin/behandling/aktivitetsplikt/AktivitetspliktService.kt

```diff
@@ -850,25 +851,48 @@ class AktivitetspliktService(
     }
 
     fun opprettOppgaveHvisIkkeVarigUnntak(dto: OpprettOppgaveForAktivitetspliktDto): OpprettOppgaveForAktivitetspliktResponse {
```

Comment:
skal vi legge til litt tester for denne?

Reply:
Ja, hvis du ser at det ikke er god dekning nå er det sikkert ikke dumt å ha litt tester på logikken rundt når vi oppretter oppgaver

Reply:
https://github.com/navikt/pensjon-etterlatte-saksbehandling/pull/7061